add PlebStopThreshold to glossary (#1638)
zkSNACKs/WasabiDoc: docs/glossary/Glossary-PrivacyWasabi.md

Peers in our documentation refers mainly to Bitcoin and Wasabi Wallet users, but it also means people.
They are literally peers in the network, or in the CoinJoin.

### Pleb Stop Threshold

> The Pleb Stop Threshold is the amount at which the wallet will not automatically start coinjoin, or it will stop coinjoin when the wallet balance is below this amount.

Reviewer: The auto coinjoin won't start if the total wallet balance is less than the plebstop.

```suggestion
The Pleb Stop Threshold is the minimum wallet balance at which coinjoin will automatically start.
```

Reviewer 2: This is called "Auto-start coinjoin threshold" in the wallet UI, shouldn't the docs match for search purposes even if the internal name is "Pleb Stop"?

Author: could. 

As the _auto-start coinjoin threshold_ is in the UI with a tooltip explainer
![image](https://github.com/zkSNACKs/WasabiDoc/assets/93143998/f7594d00-c49f-47aa-bf85-0ecea5fe5cce)
I thought explaining plebstop makes more sense, as that verb gets used sometimes (in marketing?) but it's not explained anywhere

Reviewer: `Auto-start Coinjoin (Pleb Stop) Threshold` is good enough.
